data-jdbc/src/test/resources/application-oracle-jsonview.yml

```diff
@@ -18,6 +18,6 @@ datasources:
 test-resources:
   containers:
     oracle:
-      image-name: gvenzl/oracle-free:latest-faststart
```

Comment:
JsonView began to fail with 23.3 oracle-free docker image with the error

>Caused by: Error : 1804, Position : 0, SQL = BEGIN INSERT INTO "STUDENT_VIEW" VALUES (:1 ) RETURNING JSON_VALUE(DATA,'$.id') INTO :2 ; END;, Original SQL = BEGIN INSERT INTO "STUDENT_VIEW" VALUES (?) RETURNING JSON_VALUE(DATA,'$.id') INTO ?; END;, Error Message = ORA-01804: failure to initialize timezone information |  
-- | --
  | at app//oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:710)

had to lock to 23.2 in order to fix the tests